blocks/table-of-contents/src/script.js

```diff
@@ -233,6 +243,20 @@ function getRegexFunction( regexPattern ) {
 	}
 }
 
+/**
+ * Get Append Text
+ *
+ * @param {String} regexAppend
+ * @param {String} textContent
+ * @return {String}
+ */
+function getAppendText( regexAppend, textContent ) {		
+	if( isNonEmptyString( regexAppend ) ) {
+		return (new RegExp( regexAppend, 'mi' )).exec( textContent );
+	}
+	return textContent
```

Comment:
shouldn't this return an empty string if there's no regexAppend?

Reply:
You're right, not sure what I was thinking here